lassen/asm.py

```diff
@@ -73,9 +73,34 @@ def fgetmant ():
 def fp_add(ra_mode=Mode.BYPASS, rb_mode=Mode.BYPASS):
     return inst(ALU.FP_add, ra_mode=ra_mode, rb_mode=rb_mode)
 
+def fp_sub(ra_mode=Mode.BYPASS, rb_mode=Mode.BYPASS):
+    return inst(ALU.FP_sub, ra_mode=ra_mode, rb_mode=rb_mode)
+
 def fp_mult(ra_mode=Mode.BYPASS, rb_mode=Mode.BYPASS):
     return inst(ALU.FP_mult, ra_mode=ra_mode, rb_mode=rb_mode)
 
+def fp_cmp(cond):
```

Comment:
Why not have a dedicate fp_cmp instruction so we don't have to set the Z strangely on sub

Reply:
Agreed. fixed